kuma/javascript/src/payments/pages/management.jsx

```diff
@@ -0,0 +1,188 @@
+// @flow
+import * as React from 'react';
+import { useContext, useEffect, useState } from 'react';
+
+import { gettext, interpolate, Interpolated } from '../../l10n.js';
+import UserProvider, { type UserData } from '../../user-provider.jsx';
+import Subheader from '../components/subheaders/index.jsx';
+import CancelSubscriptionForm from '../components/cancel-subscription-form.jsx';
+import ErrorMessage from '../components/error-message.jsx';
+import { getSubscriptions } from '../api.js';
+import { formatDate } from '../formatDate.js';
+
+export type SubscriptionData = {
+    id: string,
+    amount: number,
+    brand: string,
+    expires_at: string,
+    last4: string,
+    zip: string,
+    next_payment_at: string,
+};
+
+type Props = {
+    locale: string,
+};
+export const title = gettext('Manage monthly subscriptions');
+export const successMsg = gettext(
+    'Your monthly subscription has been successfully canceled.'
+);
+
+const ManagementPage = ({ locale }: Props) => {
+    const [showForm, setShowForm] = React.useState<boolean>(false);
+    const [status, setStatus] = React.useState<'success' | 'error' | 'idle'>(
+        'idle'
+    );
+    const [subscription, setSubscription] = useState<?SubscriptionData>(null);
+    const [isSubscriber, setIsSubscriber] = useState<?boolean>(null);
+    const userData: ?UserData = useContext(UserProvider.context);
+
+    // Currently we don't have a way to update the UserProvider context, so
+    // we are saving the context value `isSubscriber` to local state. It will be
+    // out-of-sync with context upon a successful delete, but I think it's ok
+    // for now, since actions after deleting are limited-- nothing builds upon
+    // `isSubscriber`. The only thing you can do from this point is go to
+    // /payments which will refresh the UserProvider context.
```

Comment:
Thanks Peter, I adopted your suggestion! 

Looking forward, I'd still prefer if we had a way to update context so it accurately reflects `isSubscriber`. I looked into it briefly but it would be a change bigger than I feel comfortable with in this PR, since `UserProvider` is used in many components. Probably not worth it just for this one page-- maybe one day if we find we need to update context in more than one place.